examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -100,10 +108,28 @@ int AppTask::Init()
 
     LightingMgr().SetCallbacks(ActionInitiated, ActionCompleted);
 
+    PlatformMgr().AddEventHandler(AppTask::ThreadProvisioningHandler, 0);
```

Comment:
Move to StartNFCTag() and call RemoveEventHandler() in ThreadProvisioningHandler?